gossipd/gossip_wire.csv

```diff
@@ -43,6 +43,13 @@ gossip_peer_connected,,lfeatures,lflen*u8
 gossip_peer_already_connected,3015
 gossip_peer_already_connected,,id,struct pubkey
 
+# gossipd -> master: attempted to connect, unsuccessful, gave up
+gossip_peer_connection_failed,3020
```

Comment:
We need some good way to allocate message numbers, now I need to bump the message number in #638 again...